pyproject.toml

```diff
@@ -12,7 +12,7 @@ include = ['poetry.lock']
 [tool.poetry.dependencies]
 python = "^3.9"
 pandas = "^1.3.2"
-pureskillgg-dsdk = ">=0.14.0"
+pureskillgg-dsdk = "^0.27.0"
```

Comment:
Can you make sure it uses the same >= format as before? This won't be an issue after v1.

Reply:
kk. I just used `poetry add pureskillgg-dsdk@latest` to update so the actual format was left to poetry.

Reply:
Should be fix'd

Reply:
Yea, it's just the default. However you can't just update the version in pyproject only, because then poetry will complain about things being out of sync.

Either have to use poetry add with the version string you want, or run poetry update after changing the project file.

Reply:
kk i did a `poetry update`